project/Common.scala

```diff
@@ -1,35 +1,37 @@
 import de.heikoseeberger.sbtheader.HeaderPlugin.autoImport._
 import de.heikoseeberger.sbtheader._
 import org.scalafmt.sbt.ScalafmtPlugin.autoImport._
+import xerial.sbt.Sonatype.autoImport._
+import sbtdynver.DynVerPlugin.autoImport._
 import sbt.Keys._
 import sbt._
 import sbt.plugins.JvmPlugin
-import sbtrelease.ReleasePlugin.autoImport.releaseCrossBuild
 
 object Common extends AutoPlugin {
 
   override def trigger = allRequirements
 
   override def requires = JvmPlugin && HeaderPlugin
 
-  override def globalSettings = Seq(
-    organization := "com.lightbend.akka",
-    organizationName := "Lightbend Inc.",
-    organizationHomepage := Some(url("https://www.lightbend.com/")),
-    startYear := Some(2016),
-    homepage := Some(url("https://akka.io")),
-    apiURL := Some(url(s"https://doc.akka.io/api/akka-persistence-cassandra/${version.value}")),
-    scmInfo := Some(
-        ScmInfo(
-          url("https://github.com/akka/akka-persistence-cassandra-1.x"),
-          "git@github.com:akka/akka-persistence-cassandra-1.x.git")),
-    developers += Developer(
-        "contributors",
-        "Contributors",
-        "https://gitter.im/akka/dev",
-        url("https://github.com/akka/akka-persistence-cassandra-1.x/graphs/contributors")),
-    licenses := Seq(("Apache-2.0", url("https://www.apache.org/licenses/LICENSE-2.0"))),
-    description := "A Cassandra plugin for Akka Persistence.")
+  override def globalSettings =
+    Seq(
+      organization := "com.lightbend.akka",
+      organizationName := "Lightbend Inc.",
+      organizationHomepage := Some(url("https://www.lightbend.com/")),
+      startYear := Some(2016),
+      homepage := Some(url("https://akka.io")),
+      apiURL := Some(url(s"https://doc.akka.io/api/akka-persistence-cassandra/${version.value}")),
+      scmInfo := Some(
+          ScmInfo(
+            url("https://github.com/akka/akka-persistence-cassandra-1.x"),
```

Comment:
this shouldn't be -1.x should it?

Reply:
Yup. Leftovers from work in previous repo. I see that in more places as well. Will fix.